src/views/Students/StudentComponents/AlumniServiceForm.jsx

```diff
@@ -230,6 +278,42 @@ const AlumniServiceForm = (props) => {
                       required={feeFieldsRequired}
                     />
                   </div>
+                  <div className="col-md-6 col-sm-12 mt-2">
+                    <Input
+                      name="program_mode"
+                      label="Program Mode"
+                      placeholder="Program Mode"
+                      control="lookup"
+                      icon="down"
+                      className="form-control"
+                      options={programOptions}
+                    />
+                  </div>
+                  <div className="col-md-6 col-sm-12 mt-2">
+                    <Input
+                      name="category"
+                      label="Category"
+                      placeholder="category"
+                      control="lookup"
+                      icon="down"
+                      className="form-control"
+                      options={categoryOptions} 
+                      onChange={(e) => setSelectedCategory(e.value)}
+                    />
+                  </div>
+                  {selectedCategory ?
+                    <div className="col-md-6 col-sm-12 mt-2">
+                      <Input
+                        name={subCategoryName}
+                        label={subCategoryLabel}
+                        placeholder={subCategoryLabel}
+                        control="lookup"
+                        icon="down"
+                        className="form-control"
+                        options={subCategoryOptions} 
+                      />
+                    </div> 
+                  : null }
```

Comment:
A short version for same thing:
```suggestion
                  {selectedCategory &&
                    <div className="col-md-6 col-sm-12 mt-2">
                      <Input
                        name={subCategoryName}
                        label={subCategoryLabel}
                        placeholder={subCategoryLabel}
                        control="lookup"
                        icon="down"
                        className="form-control"
                        options={subCategoryOptions} 
                      />
                    </div> 
                  }
```